packages/portal/docs/style/BootstrapVue/Alert.md

```diff
@@ -0,0 +1,21 @@
+Customised BootstrapVue alert component.
+
+Docs including more features and variants: https://bootstrap-vue.org/docs/components/alert
+
+Used by:
+- AlertMessage
+- InfoMessage
+- SearchFilters (Newspaper/WWI api filter alert)
+
+Default variant
```

Comment:
Only one of them is default. Either remove this line, or have two code blocks, the second with its own "dark variant" heading.

Reply:
Removed the line.